Task-4/js/index.js

```diff
@@ -0,0 +1,72 @@
+function validFormFieldInput(event) {
```

Comment:
. Implement a JavaScript function named `validFormFieldInput(data)`
3. Add an ID attribute to each form field and implement the code needed to retrieve the each form field value using the following method:
      ```javascript
       const newTaskNameInput = document.querySelector('#newTaskNameInput');
       const name = newTaskNameInput.value;

Reply:
I believe using querySelector includes the use of IDs also